### PR TITLE
[WIP] Update Translations Guide.

### PR DESCRIPTION
**Pull request by raisedadead**
### DON'T MERGE

Check : https://github.com/raisedadead/wiki/blob/update/translations-guide/Translations-Guide.md
Refers #565

### CODE REVIEW

**Comment by Rafase282** [comment]
What's the progress on this?

**Comment by raisedadead** [comment]
@Rafase282 Thanks for the reminder, and apologies for the delay. 
I will try complete this by tomorrow.

Only thing that would remain not resolved is the Google Translate process. But that can be taken up in a separate PR.

**Comment by theflametrooper** [comment]
A couple of typos I found...

Line 120 -  _Glosary_ should be **Glossary**
Line 9 - _below work flow_ should be **workflow below**.

There are probably other spelling mistakes that will need checked before being merged.

File: Translations-Guide.md [hunk 1]
```diff
@@ -1,29 +1,88 @@
...
 
-## 3. Issues related with the translation
-In the [issues section](https://github.com/FreeCodeCamp/FreeCodeCamp/issues) of FreeCodeCamp there can be 2 kind of requests related with the translation of the challenges:
-- Some to start the translation of one of the files in the directory `seeds/challenges`. See for example: [Issue #4544] ([https://github.com/FreeCodeCamp/FreeCodeCamp/issues/4544](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/4544))   
-- Other issues requesting to verify or to update an existing translation of one the files in directory `seeds/challenges`. For example: [https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6517](https://github.com/FreeCodeCamp/FreeCodeCamp/issues/6517) These could be reopened for example when there is a change in the English version of the challenge.
+<img src="./images/Translations-Guide/Review_Comment.gif" alt="Review Comment">
+
+## Create `Translation` issues to track the progress.
+
+Before starting translations of challenges GitHub issues need to be created specific to the language of translation. Translators should create issues with the details of the challenges to be tracked in that particular issue.
```
Comment (theflametrooper) [inline Translations-Guide.md]: Comma needed after **Before starting translations of challenges**

File: Translations-Guide.md [hunk 2]
```diff
@@ -1,29 +1,88 @@
...
 
-## 1. Chat room
-We recommend you to create a chat room in gitter.im exclusive for the language that you're working on, communicate there in that language, discuss the current the status of the project and coordinate efforts. Help regarding the technical process should also be provided in the chat room. Many people want to collaborate, but get intimidated by the learning curve of Git. While it's not difficult to get started, it's always nice to have someone to help you if you get stuck. Many users go to the official translators chat room of FreeCodeCamp at [**`[FreeCodeCamp/Translators]`**](https://gitter.im/FreeCodeCamp/Translators) . You can go there to recruit new translators for your language every once in a while.
+Thank you so much for your interest in translating FreeCodeCamp. Reading this document is recommended for participating in a collective effort to bring FreeCodeCamp to more and more people all around the world.
+
+## How to Contribute to Translations?
+
+There are various ways in which you can contribute collaboratively to the translations. Every translation effort usually follows the below work flow.
```
Comment (theflametrooper) [inline Translations-Guide.md]: _below work flow_ should be **workflow below**.

File: Translations-Guide.md [hunk 3]
```diff
@@ -54,27 +113,29 @@ Take into account that this file has been created as we work on the translation
...
 - Try not to be too formal yet not too casual, just to keep things friendly.
 - In order to make contents more understandable for native speakers of your target language (think of those who don't speak English), translate as much as you can, try to use a word in English only if it is already widely used in the countries where your target language is spoken.
 
-## 6. Glosary
-It is efficient if all the translators working on the same language create a glossary showing the translation of words in English employed in FreeCodeCamp's challenges. Sometimes there is more than one way of translating some terms, and regional differences might create confusion (for example, some terms might differ between Spanish from Spain and from Latin America, or between the French language employed in Canada and in France).  Be democratic! Choose the most appropriate translation by voting and keep a record of the results. One example of such record can be found here: [FreeCodeCamp Glosary (English to Spanish)](https://docs.google.com/spreadsheets/d/1c60Sl4MAAsZ7biCPgur7A4aVqhErIfwrE1SulPqbOGo/edit#gid=0) Use the chat room for discussing the glossary, so no one will miss anything.
+### Glosary
```
Comment (theflametrooper) [inline Translations-Guide.md]: _Glosary_ Should be **Glossary**

**Comment by abisz** [comment]
The guide suggests "add/" as a branch prefix, but the camperbot only accepts "translate/" (which I think is a better approach). Maybe change it in the guide?

**Comment by alayek** [comment]
@raisedadead what's the progress on this?

File: Translations-Guide.md [hunk 4]
```diff
@@ -1,115 +1,179 @@
...
-If it doesn't exist please let us know in the 
-[Translators channel](https://gitter.im/FreeCodeCamp/Translators), 
-or create one yourself and share it at the  
-[translators channel](https://gitter.im/FreeCodeCamp/Translators) 
-and at other places to invite more translators and reviewers that speak your 
-language.  
+- [x] Discuss in language specific Translations Chat Rooms.
+- [x] Review the work of another translator.
+- [x] Create `Translation` issues to track the progress.
+- [x] Do Translations.
```
Comment (theflametrooper) [inline Translations-Guide.md]: Maybe change this to "Implement Translations" or "Complete Translations"

File: Translations-Guide.md [hunk 5]
```diff
@@ -1,115 +1,179 @@
...
+
+If you see issues already open for the challenges, then you could proceed to working on them.
+
+OR
+
+If you can't see any related issues, or that can't be reopened, then start by __creating__ them for the challenge category you would like work on. Remember to discuss in the Chat room for a confirmation of the challenges that you would like to add to the issue.
+
+> _Pro Tip : Follow the example shown above, for consistency in the formatting of the issues, like adding checklists for challenges._
+
+## Do Translations.
```
Comment (theflametrooper) [inline Translations-Guide.md]: Change this to "Implement Translations" or "Complete Translations"

**Comment by Rafase282** [comment]
@raisedadead Could you add a bit about filenames? Basically they can't be the same filename even if they are in a different directory, this means that they will either have to prefix the language or rename it in the target language. We should discuss this further in the wiki chat.

**Pull request merged**